lib/Dialect/HW/HWOps.cpp

```diff
@@ -2294,6 +2304,11 @@ OpFoldResult ArrayGetOp::fold(ArrayRef<Attribute> operands) {
   if (!inputCreate)
     return {};
 
+  // If the array has a single uniform value for each element, just return that
```

Comment:
Nit: please update comment above (on this method) to reflect this change.

Reply:
Done